src/arithmetic/canonical.cc

```diff
@@ -532,18 +534,22 @@ class Canonical::Internal : public IRMutator {
       return make_zero(t);
     }
   }
+public:
```

Comment:
this is not necessary since we are already in public

Reply:
fair enough. thx.